source/style-guide/markup/format/lists.txt

```diff
@@ -0,0 +1,107 @@
+=====
+Lists
+=====
+
+.. default-domain:: mongodb
+
+.. include:: /includes/styles/corrections.rst
+
+Each level of a list must start and end with a blank line.
+
+Unordered Lists
+~~~~~~~~~~~~~~~
+
+- Use hyphens (``-``) to indicate items in an unordered list. 
+- Use the hyphen at every level of a multi-level unordered list.
```

Comment:
I think this can be removed since it duplicates information presented in the second sentence of the following item.